about.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>About</title>
+</head>
+<body>
+    <header class="header1">
+        <div class="header1-content">
+            <ul>
+                <li>English</li>
+                <li>My Page</li>
+                <li>Logout</li>
+           </ul>
+        </div>
+    </header>
+    <img src="img/close.png" width="40" height="40" alt="close-btn" class="close-btn">
+    <header>
+        <nav class="navbar" id="navbar-list">
+            <ul>
+                <li id="logo"><a href="index.html"><img src="img/logo.jpg" alt="logoimage" width="130" height="50"></a>
+                </li>
+                <li><a href="about.html" rel="noreferrer noopener">About</a></li>
+                <li><a href="#" rel="noreferrer noopener">Program</a></li>
+                <li><a href="#" rel="noreferrer noopener">Join</a></li>
+                <li><a href="#" rel="noreferrer noopener">Sponsor</a></li>
+                <li><a href="#" rel="noreferrer noopener">News</a></li>
+                <li id="websummit"><a href="#" rel="noreferrer noopener">Web Summit 2021</a></li>
+            </ul>
+        </nav>
+    </header>
+    <section class="about" id="about1">
+        <img src="img/menu.png" width="40" height="40" alt="hamburger-button" class="hamburger-button"
+            id="hamburger-btn2">
+        <div class="about-main">
+            <h2>"Hello,Tech World"</h2>
+            <h1>Technology Conference<br>Web Summit<br>2021</h1>
+        </div>
+        <div class="about-description">
+            <p>We’re a company from Dublin, Ireland, that holds events across the world: Web Summit in Lisbon and Tokyo,
+                Collision in Toronto, and RISE in Hong Kong.</p>
+        </div>
+        <div class="about-contact-us">
+            <h2>Please visit the support center for any further questions about the Technology Web Summit 2021</h2>
+            <a href="https://support.websummit.com/support/home" rel="noreferrer noopener">https://support.websummit.com/support/home</a>
+        </div>
+        <div class="logo">
+            <h2>Technology Conference<br>Web Summit 2021 Logo</h2>
+            <h3>The logo of Technology Conference Web Summit logo was designed by <a href="https://websummit.com/"
+                    target="_blank" rel="noreferrer noopener">Web Sumit</a> </h3>
+            <h3>For more info please visit this link</h3><a
+                href="https://commons.wikimedia.org/wiki/File:Logo_Lisbon_Web_Summit.svg" target="_blank" rel="noreferrer noopener"> here</a>
+        </div>
+        <img src="img/logo.jpg" alt="logoimage" class="logo-image">
```

Comment:
-[ ] Make border in this case bigger so it will look more like the originial
![Screenshot from 2021-10-13 19-49-12](https://user-images.githubusercontent.com/62705947/137186326-1e35f5f6-fbde-444d-8e33-ce0049201005.png)

Reply:
This issue is well resolved! ✔️